<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="letter_buttons_container" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>571</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#letter_buttons_container{
	border: gray solid 1px;
}

#letter_buttons_container  *{
	background-color: yellow;
}

#letter_buttons_container *[class=&quot;active&quot;]{
	border: 5px solid blue;
}</string>
    </property>
    <widget class="QPushButton" name="pushButton_letter_z">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>110</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_n">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_y">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_d">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>D</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_h">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_f">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_q">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Q</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_m">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>M</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_r">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_s">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_c">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_i">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>I</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_t">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>T</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_a">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_w">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>W</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_o">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>O</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_k">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>K</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_u">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>U</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_l">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>L</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_g">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>G</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_b">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_x">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_p">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_v">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>V</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_j">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>J</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_letter_e">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>E</string>
     </property>
     <property name="class" stdset="0">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>571</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password_guess">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_guess_password">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>guess password</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>290</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ranking</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>50</y>
       <width>351</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>true</bool>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_letter">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vote for character</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
